app/src/main/java/com/github/k409/fitflow/ui/screen/friends/FriendCard.kt

```diff
@@ -0,0 +1,346 @@
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .clip(RoundedCornerShape(8.dp))
+            .background(MaterialTheme.colorScheme.primaryContainer)
+            .padding(horizontal = 8.dp, vertical = 2.dp),
+    ) {
+        Row(
+            modifier = Modifier
+                .fillMaxSize(),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.spacedBy(4.dp),
+        ) {
+            SubcomposeAsyncImage(
+                model = ImageRequest.Builder(LocalContext.current).data(user.photoUrl)
+                    .crossfade(true)
+                    .build(),
+                contentDescription = null,
+                contentScale = ContentScale.Crop,
+                error = {
+                    Icon(
+                        modifier = Modifier.padding(3.dp),
+                        imageVector = Icons.Outlined.PersonOutline,
+                        contentDescription = null,
+                    )
+                },
+                modifier = Modifier
+                    .clip(CircleShape)
+                    .size(80.dp)
+                    .background(
+                        color = MaterialTheme.colorScheme.secondaryContainer,
+                        shape = CircleShape,
+                    )
+                    .border(
+                        width = 2.dp,
+                        color = MaterialTheme.colorScheme.onSurface.copy(alpha = 0.12f),
+                        shape = CircleShape,
+                    ),
+            )
+
+            Spacer(modifier = Modifier.width(2.dp))
+
+            Column(
+                modifier = Modifier.fillMaxSize(),
+            ) {
+                Text(
+                    modifier = Modifier
+                        .clip(MaterialTheme.shapes.medium)
+                        .padding(4.dp),
+                    text = user.name,
+                    fontWeight = FontWeight.Bold,
+                    style = MaterialTheme.typography.bodyLarge,
+                    color = MaterialTheme.colorScheme.primary,
+                )
+
+                Text(
+                    modifier = Modifier
+                        .clip(MaterialTheme.shapes.medium)
+                        .padding(4.dp),
+                    text = user.email,
+                    style = MaterialTheme.typography.bodyMedium,
+                    color = MaterialTheme.colorScheme.primary,
+                )
+
+                Button(
+                    modifier = Modifier
+                        .padding(start = 6.dp),
+                    colors = ButtonDefaults.buttonColors(MaterialTheme.colorScheme.error),
+                    onClick = {
+                        coroutineScope.launch {
+                            friendsViewModel.removeFriend(user.uid)
+                        }
+                        SnackbarManager.showMessage(context.getString(R.string.friend_removed))
+                    },
```

Comment:
Maybe add confirmation dialog (like in market purchases) if user really wants to remove friend.